src/SanityCheck.cpp

```diff
@@ -120,6 +121,7 @@ void SanityCheck::getDefaultDirection(vector<vector<double> >& w) {
       }
     } else {
       // I want to assign the default vector that is present in the input file
+      #pragma omp parallel for schedule(dynamic, 1)
```

Comment:
Why do we need OMP here? This seems like a very cheap operation.

Reply:
Finding the initial direction was taking a bit of time for really large pin files and I was checking whether it could be sped up just by running the direction search in parallel.  I removed the OMP call in the last commit